SPECS/expat/expat.spec

```diff
@@ -65,6 +67,9 @@ rm -rf %{buildroot}/%{_docdir}/%{name}
 %{_libdir}/libexpat.so.1*
 
 %changelog
+* Tue Mar 19 2024 Alejandro Martinez <alejandroma@microsoft.com> 2.5.0-2
```

Comment:
You missed incrementing the `Release` number. Since `expat` is a toolchain package, you'll also need to update the manifest files under `toolkit/resources/manifests/package/*.txt`.